contracts/deploy/018_upgrade_governor.js

```diff
@@ -0,0 +1,235 @@
+// 1. Deploy the latest governor contract and sets its ownership to the new multisig safe wallet.
+// 2. Submit a proposal on the old governor to call transferGovernance() on all governable contracts.
+// 3. Submit a proposal on the new governor to call claimGovernance() on all governable contracts.
+
+const {
+  isMainnet,
+  isFork,
+  isRinkeby,
+  isSmokeTest,
+} = require("../test/helpers.js");
+const {
+  log,
+  deployWithConfirmation,
+  withConfirmation,
+  executeProposal,
+  sendProposal,
+} = require("../utils/deploy");
+const { proposeArgs } = require("../utils/governor");
+const { getTxOpts } = require("../utils/tx");
+const addresses = require("../utils/addresses");
+
+const deployName = "018_upgrade_governor";
+
+const runDeployment = async (hre) => {
+  console.log(`Running ${deployName} deployment...`);
+
+  const { governorAddr } = await hre.getNamedAccounts();
+
+  // Signers
+  const sGovernor = await ethers.provider.getSigner(governorAddr);
+
+  const cOUSDProxy = await ethers.getContract("OUSDProxy");
+  const cVaultProxy = await ethers.getContract("VaultProxy");
+  const cCompoundStrategyProxy = await ethers.getContract(
+    "CompoundStrategyProxy"
+  );
+  const cThreePoolStrategyProxy = await ethers.getContract(
+    "ThreePoolStrategyProxy"
+  );
+  const cAaveStrategyProxy = await ethers.getContract("AaveStrategyProxy");
+  const cBuyback = await ethers.getContract("Buyback");
+  const cOGNStakingProxy = await ethers.getContract("OGNStakingProxy");
+  const cCompensationClaim = await ethers.getContract("CompensationClaims");
+
+  // Deploy a new governor contract.
+  // The governor's admin is the guardian account (e.g. the multi-sig).
+  // Set a min delay of 60sec for executing proposals.
+  const dGovernor = await deployWithConfirmation("Governor", [
```

Comment:
Just making sure that the deploy on this line immediately changes the the result of getContract("Governor"), as used by sendProposal?

Reply:
yes, I double checked and it does.